app/models/job/Job.scala

```diff
@@ -141,7 +141,9 @@ class JobDAO @Inject()(sqlClient: SqlClient)(implicit ec: ExecutionContext)
      """
 
   private def listAccessQ(requestingUserId: ObjectId) =
-    q"""_owner = $requestingUserId"""
+    q"""_owner = $requestingUserId OR
+       ((SELECT u._organization FROM webknossos.users_ u WHERE u._id = _owner) IN (SELECT _organization FROM webknossos.users_ WHERE _id = $requestingUserId AND isAdmin))
```

Comment:
Hey, I know that the review was already done by @fm3 

But to me the added SQL reads kinda strange. How I read it is that you 
> `(SELECT u._organization FROM webknossos.users_ u WHERE u._id = _owner)`

Select the organization id by reading it from the user table from he column where the user is the actual owner of the organization in question. And then 
> `(SELECT _organization FROM webknossos.users_ WHERE _id = $requestingUserId AND isAdmin)`

you check whether the current user (`requestingUserId`) is in the same organization and is an admin? 

To me that sounds like this can be simplyied into a single select statement and not a `((Select ...) in (Select...))` statement which reads much more complex.

I'd say that 
```
(SELECT true FROM webknossos.users_ u WHERE u._id = _owner AND u._id = $requestingUserId AND isAdmin)
```
should evaluate to the same request on a logical level.

But I also might be wrong here ... :thinking:

Reply:
To me it seems like your query seems to check that the requesting user is the owner, which is what should not be needed since the admin should also be able to view it.

Reply:
Oh ah I see. The first subquery is needed to identify the orga id as only the owner id is know and the second checks whether the user is in the same orga and an admin :+1:

In that case it looks good :) => Nvm my mistake :see_no_evil: